crates/project/src/terminals.rs

```diff
@@ -277,6 +280,20 @@ impl Project {
         settings: &TerminalSettings,
         cx: &AppContext,
     ) -> Option<PathBuf> {
+        if let Some((worktree, _)) = self.find_worktree(abs_path, cx) {
+            let toolchain = self
+                .active_toolchain(worktree.read(cx).id(), LanguageName::new("Python"), cx)
+                .now_or_never()
+                .unwrap();
+
+            if let Some(toolchain) = toolchain {
+                let mut path = PathBuf::from(String::from(toolchain.path));
```

Comment:
Minor nit, but you can probably turn `toolchain.path` into &str instead of going through String. That should maybe allow us to skip a single allocation, though it's not a big deal really.
Feel free to ignore.